### PR TITLE
Update package SKU and version as part of release process

### PR DESCRIPTION
**Pull request by tnorling**
Adds a script that copies `name` and `version` properties from `package.json` into a new file `src/version.json` during the prepack phase of release. This allows us to use this data in the library without emitting package.json in the released package and resolves tsdx warnings about setting `rootDir` to `./src`

### CODE REVIEW

File: lib/msal-browser/src/version.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"name":"@azure/msal-browser","version":"2.9.0"}
```
Comment (pkanher617) [inline lib/msal-browser/src/version.json:1]: Is there a reason we can't just take the version from the package.json?

File: lib/msal-common/package.json [hunk 2]
```diff
@@ -54,7 +54,8 @@
     "build:modules:watch": "rollup -cw",
     "build": "npm run clean && npm run build:modules",
     "build:all": "npm run build",
-    "prepack": "npm run build"
+    "prepack": "npm run updateVersion && npm run build",
+    "updateVersion": "node ../../release-scripts/updateVersion.js"
```
Comment (pkanher617) [inline lib/msal-common/package.json:58]: Would it be better to run this after `beachball:bump`? Or a combination of the two.

File: lib/msal-browser/src/version.json (hunk 1 above)
Comment (tnorling) [reply lib/msal-browser/src/version.json:1]: We shouldn't be emitting things in the build that live outside the root directory and some tools like tsdx require that the root directory is set to "./src". Though this isn't causing problems for browser today it is for other libs so the change here is mainly to keep all our libraries aligned

File: release-scripts/updateVersion.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+const { name, version } = require(packageJsonPath);
+const fs = require("fs");
+
+const data = {
+    name: name,
+    version: version
+};
+
+const outputFilepath = path.resolve(process.cwd(), "src", "version.json");
+fs.writeFileSync(outputFilepath, JSON.stringify(data));
```
Comment (jasonnutter) [inline release-scripts/updateVersion.js:17]: Can you add some basic success and error handling?

File: lib/msal-common/package.json (hunk 2 above)
Comment (tnorling) [reply lib/msal-common/package.json]: Done

File: release-scripts/updateVersion.js (hunk 3 above)
Comment (tnorling) [reply release-scripts/updateVersion.js]: Done

**Review by jo-arroyo — APPROVED** [review]

**Review by jasonnutter — APPROVED** [review]

**Review by hectormmg — APPROVED** [review]

**Pull request merged**